src/main/java/core/basesyntax/dao/StorageDao.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.dao;
+
+import java.util.List;
+import java.util.Map;
+
+public interface StorageDao {
+    void putFruit(String key, Integer value);
+
+    List<String> getAllFruits();
+
+    Map<String, Integer> getMap();
```

Comment:
Давай тут переробимо метод щоб не передавати всю мапу. Бо він використовується якраз тоді, коли нам потрібна кількість одного фрукту
```suggestion
    Integer getFruitAmount(String fruitName);
```